### DISCUSSION TITLE
Symbols not visible

### DISCUSSION THREAD

**Discussion by ship51** [sumatrapdfreader/sumatrapdf#2518] [Ideas]
I recently observed that Sumatra was not displaying the symbols such as `+, -,*,/ ,( ,) , epsilon`, etc. when I looked up the actual book of the eBook I had.
It seems it recognizes it is some character but is unable to read it. Any suggestions?

**eBook** 
![SumatraPDF_PueOH4WdpD](https://user-images.githubusercontent.com/57025785/166102399-e8228df9-df9c-41d4-be71-e4d55fba47c0.png)

**Actual book**
![photo6298736949275505037](https://user-images.githubusercontent.com/57025785/166102423-6625e43b-7fc3-4ee5-ba59-daca99662d35.jpg)

**Comment by kjk** [comment]
Does it happen in [pre-release](https://www.sumatrapdfreader.org/prerelease)?

If yes, please [open an issue](https://github.com/sumatrapdfreader/sumatrapdf/issues) and provide a document showing a problem.

There's nothing I can do or say without access to file that shows the problem.
